docs/installation/server-config/security.rst

```diff
@@ -0,0 +1,163 @@
+.. _ksql-security:
+
+Securing KSQL
+-------------
+
+To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. In the same way,
+KSQL is configured so that the KSQL servers are authenticated and authorized, and data communication is encrypted when
+communicating with the Kafka cluster. KSQL can be configured for:
+
+- :ref:`SSL for encryption <kafka_ssl_encryption>`
+- :ref:`SASL for authentication <kafka_sasl_auth>`
+- :ref:`HTTPS for Confluent Schema Registry <schema_registry_http_https>`
+
+-------------------------------------------------
+Using KSQL with an Apache Kafka Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I would start this section (e.g. with an info box) that points readers to our Kafka security documentation. Many readers might see stuff like ACLs for Kafka for the first time.

Same for the non-interactive section.  Or put it into the parent section.

Reply:
added `.. tip:: For more information about ACLs see :ref:`kafka_authorization` and for more information about interactive and
         non-interactive queries, see :ref:`streams_developer-guide_interactive-queries`.`

Reply:
updated to link to `restrict-ksql-interactive` (e.g. https://github.com/confluentinc/ksql/blob/8cfd40a4de62222923c45a436f50c15c4a8d7582/docs/installation/server-config/index.rst).